lib/src/shared_model/item_location_override.dart

```diff
@@ -5,29 +5,33 @@ import 'package:square_connect/square_connect.dart';
 part 'item_location_override.g.dart';
 
 @JsonSerializable()
-class ItemVariationLocationOverride extends Equatable {
-  const ItemVariationLocationOverride({
+class ItemVariationLocationOverrides extends Equatable {
+  const ItemVariationLocationOverrides({
     required this.locationId,
     this.priceMoney,
     this.pricingType,
     this.trackInventory,
     this.inventoryAlertType,
     this.inventoryAlertThreshold,
+    this.soldOut,
+    this.soldOutValidUntil,
   });
 
-  /// Converts a [Map] to an [ItemVariationLocationOverride]
-  factory ItemVariationLocationOverride.fromJson(Map<String, dynamic> json) =>
-      _$ItemVariationLocationOverrideFromJson(json);
+  /// Converts a [Map] to an [ItemVariationLocationOverrides]
+  factory ItemVariationLocationOverrides.fromJson(Map<String, dynamic> json) =>
+      _$ItemVariationLocationOverridesFromJson(json);
 
-  /// Converts a [ItemVariationLocationOverride] to a [Map]
-  Map<String, dynamic> toJson() => _$ItemVariationLocationOverrideToJson(this);
+  /// Converts a [ItemVariationLocationOverrides] to a [Map]
+  Map<String, dynamic> toJson() => _$ItemVariationLocationOverridesToJson(this);
 
   final String locationId;
   final Money? priceMoney;
   final CatalogPricingType? pricingType;
   final bool? trackInventory;
   final InventoryAlertType? inventoryAlertType;
   final int? inventoryAlertThreshold;
+  final bool? soldOut;
+  final String? soldOutValidUntil;
```

Comment:
```suggestion
  final DateTime? soldOutValidUntil;
```
Shouldn't this be a timestamp?

Reply:
https://github.com/mtwichel/square-connect-flutter-library/blob/f69ad99cef67d8bd74e6c685d7fd861545ef0e44/lib/src/shared_model/customer.dart#L61

Reply:
> And it looks like that we have just been doing strings in the package:
https://github.com/mtwichel/square-connect-flutter-library/blob/f69ad99cef67d8bd74e6c685d7fd861545ef0e44/lib/src/shared_model/card_payment_timeline.dart#L29

That's a bummer - it'd be cool if these were datetimes but I suppose we should keep it consistent